ui/src/components/flows/tasks/TaskString.vue

```diff
@@ -37,7 +37,7 @@
             :full-height="false"
             :should-focus="false"
             schema-type="flow"
-            lang="plaintext"
+            :lang="'python'"
```

Comment:
This text field can be python, yaml, javascript or a number of other scripting languages.
Do you think we should always syntax highlight for Python?

We could try and guess the language used (like in vscode).